### ISSUE TITLE
Error recognizing plate number in the picture

### ISSUE THREAD

**Issue by tomeroto** [sujaybhowmick/OpenAlprDroidApp#13]
I was try sample image for California license plate.
Here the image: http://www.15q.net/us1/ca14.jpg

But after I take the picture, it always fast response (likely no processing) and say:

"Error recognizing plate number in the picture"

How to fix it? What I miss something?

I use Android Studio, and Phone with OS Marsmellow.

Need help.

**Comment by ShreyashPromact** [comment]
@tomeroto 
The existing code has changed the country from **_us_** to **_ca_**. I suggest you keep **_us_** only in `onActivityResult` method and then you will be able to get the number plate detected.

Do let me know if any help needed. 
Thanks.

**Comment by fengboZ** [comment]
Hi, I want to know if this code is able to detect Thai number plate?

**Comment by fengboZ** [comment]
@ShreyashPromact

**Comment by ShreyashPromact** [comment]
@fengboZ 
I don't think it will detect the Thai number plate. This is because it is made for a specific country only,
